.github/workflows/stale-branch-sweep.yml

```
@@ -0,0 +1,176 @@
+      - name: Post orphan report to issue
+        env:
+          GH_TOKEN: ${{ secrets.ACTIONS_PAT || secrets.GITHUB_TOKEN }}
+        run: |
+          ORPHANS="${{ steps.orphans.outputs.orphans }}"
+          ORPHAN_COUNT="${{ steps.orphans.outputs.orphan_count }}"
+          ACTIVE_COUNT="${{ steps.orphans.outputs.active_count }}"
+          MODE="DRY RUN"
+          [ "${DRY_RUN}" = "false" ] && MODE="LIVE DELETE"
+
+          BODY="## Stale Branch Sweep Report
+
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected key

(syntax-check)